provider/cluster/kube/config.go

```diff
@@ -5,7 +5,8 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-type config_ struct {
+// Config is the struct that stores kube config
+type Config struct {
```

Comment:
keep this private to the package

Reply:
Yeah, changed it to private but renamed it to `kubeconfig` just to avoid warning `don't use underscores in go names`